addons/delivery/data/delivery_data.xml

```diff
@@ -24,5 +24,47 @@
             <field name="delivery_type">fixed</field>
             <field name="product_id" ref="delivery.product_product_delivery"/>
         </record>
+
+        <!-- Local Delivery -->
+        <record id="product_product_local_delivery" model="product.product">
+            <field name="name">Local Delivery</field>
+            <field name="default_code">Delivery_010</field>
+            <field name="type">service</field>
+            <field name="categ_id" ref="delivery.product_category_deliveries"/>
+            <field name="sale_ok" eval="False"/>
+            <field name="purchase_ok" eval="False"/>
+            <field name="list_price">5.0</field>
+            <field name="invoice_policy">order</field>
+        </record>
+        <record id="delivery_local_delivery" model="delivery.carrier">
+            <field name="name">Local Delivery</field>
+            <field name="fixed_price">5.0</field>
+            <field name="free_over" eval="True"/>
+            <field name="amount">50</field>
+            <field name="sequence">4</field>
+            <field name="delivery_type">fixed</field>
+            <field name="product_id" ref="delivery.product_product_local_delivery"/>
+        </record>
+
+        <!-- Local Pickup -->
+        <record id="product_product_local_pickup" model="product.product">
+            <field name="name">Local Pickup</field>
+            <field name="default_code">Delivery_011</field>
+            <field name="type">service</field>
+            <field name="categ_id" ref="delivery.product_category_deliveries"/>
+            <field name="sale_ok" eval="False"/>
+            <field name="purchase_ok" eval="False"/>
+            <field name="list_price">5.0</field>
+            <field name="invoice_policy">order</field>
+        </record>
+        <record id="delivery_local_pickup" model="delivery.carrier">
+            <field name="name">Local Pickup</field>
+            <field name="fixed_price">5.0</field>
+            <field name="free_over" eval="False"/>
+            <field name="sequence">5</field>
+            <field name="delivery_type">fixed</field>
+            <field name="product_id" ref="delivery.product_product_local_pickup"/>
+            <field name="carrier_description">Pick up goods at the following address:</field>
+        </record>
     </data>
```

Comment:
This change was moved to delivery_demo.xml instead which has a decent amount of additional lines that would need to be unindented, so I will avoid the unnecessary change diff this time. IMO It would better to do this kind of cleanup for the whole module at once in a separate dedicated PR :)